llama-index-integrations/llms/llama-index-llms-openai/llama_index/llms/openai/utils.py

```diff
@@ -250,10 +255,32 @@ def is_function_calling_model(model: str) -> bool:
 def to_openai_message_dict(
     message: ChatMessage, drop_none: bool = False, model: Optional[str] = None
 ) -> ChatCompletionMessageParam:
-    """Convert generic message to OpenAI message dict."""
+    """Convert a ChatMessage to an OpenAI message dict."""
+    content = []
+    for block in message.blocks:
+        if isinstance(block, TextBlock):
+            content.append({"type": "text", "text": block.text})
+        elif isinstance(block, ImageBlock):
```

Comment:
Should we have an else to raise an error for any other types? Just to catch ourselves in the future

Reply:
Good idea 👍